Oh, indeed! And it fails to build exactly because `certifi` is not bumped
https://hydra.nixos.org/build/75612592/nixlog/1/tail

```
unpacking sources
unpacking source archive /nix/store/83lvqn2nn8n5il6k8h6jm4d4kh9a9r45-elasticsearch-curator-5.5.4.tar.gz
source root is elasticsearch-curator-5.5.4
setting SOURCE_DATE_EPOCH to timestamp 1527091381 of file elasticsearch-curator-5.5.4/setup.cfg
patching sources
configuring
building
Download error on https://pypi.org/simple/certifi/: [Errno -2] Name or service not known -- Some packages may not be found!
Download error on https://pypi.org/simple/: [Errno -2] Name or service not known -- Some packages may not be found!
No local packages or working download links found for certifi>=2018.4.16
Traceback (most recent call last):
  File "nix_run_setup", line 8, in <module>
    exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
  File "setup.py", line 173, in <module>
    tests_require = ["mock", "nose", "coverage", "nosexcover"]
  File "/nix/store/x9yd5nwcqndjmqzpysfhl426wwpzbqz9-python2.7-bootstrapped-pip-10.0.1/lib/python2.7/site-packages/setuptools/__init__.py", line 128, in setup
    _install_setup_requires(attrs)
  File "/nix/store/x9yd5nwcqndjmqzpysfhl426wwpzbqz9-python2.7-bootstrapped-pip-10.0.1/lib/python2.7/site-packages/setuptools/__init__.py", line 123, in _install_setup_requires
    dist.fetch_build_eggs(dist.setup_requires)
  File "/nix/store/x9yd5nwcqndjmqzpysfhl426wwpzbqz9-python2.7-bootstrapped-pip-10.0.1/lib/python2.7/site-packages/setuptools/dist.py", line 514, in fetch_build_eggs
    replace_conflicting=True,
  File "/nix/store/x9yd5nwcqndjmqzpysfhl426wwpzbqz9-python2.7-bootstrapped-pip-10.0.1/lib/python2.7/site-packages/pkg_resources/__init__.py", line 770, in resolve
    replace_conflicting=replace_conflicting
  File "/nix/store/x9yd5nwcqndjmqzpysfhl426wwpzbqz9-python2.7-bootstrapped-pip-10.0.1/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1053, in best_match
    return self.obtain(req, installer)
  File "/nix/store/x9yd5nwcqndjmqzpysfhl426wwpzbqz9-python2.7-bootstrapped-pip-10.0.1/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1065, in obtain
    return installer(requirement)
  File "/nix/store/x9yd5nwcqndjmqzpysfhl426wwpzbqz9-python2.7-bootstrapped-pip-10.0.1/lib/python2.7/site-packages/setuptools/dist.py", line 581, in fetch_build_egg
    return cmd.easy_install(req)
  File "/nix/store/x9yd5nwcqndjmqzpysfhl426wwpzbqz9-python2.7-bootstrapped-pip-10.0.1/lib/python2.7/site-packages/setuptools/command/easy_install.py", line 667, in easy_install
    raise DistutilsError(msg)
distutils.errors.DistutilsError: Could not find suitable distribution for Requirement.parse('certifi>=2018.4.16')
builder for '/nix/store/a3hqi1dfjwnrwi3mpyiyklvvihqaxicg-python2.7-elasticsearch-curator-5.5.4.drv' failed with exit code 1
```

--- a/pkgs/development/python-modules/certifi/default.nix
+++ b/pkgs/development/python-modules/certifi/default.nix
@@ -5,12 +5,11 @@
 
 buildPythonPackage rec {
   pname = "certifi";
-  version = "2018.1.18";
-  name = "${pname}-${version}";
+  version = "2018.4.16";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "edbc3f203427eef571f79a7692bb160a2b0f7ccaa31953e99bd17e307cf63f7d";
+    sha256 = "13e698f54293db9f89122b0581843a782ad0934a4fe0172d2a980ba77fc61bb7";
   };
 
   meta = {

